File: app/src/main/java/ru/androidacademy/ateam/model/tables/Word.java
package ru.androidacademy.ateam.model.tables;

import android.arch.persistence.room.Entity;
import android.arch.persistence.room.PrimaryKey;

@Entity
public class Word {
    @PrimaryKey(autoGenerate = true)
   public long id;
   public String word;
   public long deckId;
}

File: app/src/main/java/ru/androidacademy/ateam/model/tables/Word.kt
package ru.androidacademy.ateam.model.tables

import androidx.room.Entity
import androidx.room.PrimaryKey


@Entity
data class Word(
    var text: String
    , var deckId: Long
) {
    @PrimaryKey(autoGenerate = true)
    var id: Long = 0
}
